### DISCUSSION TITLE
ChatGPT Extension Request or question

### DISCUSSION THREAD

**Discussion by Austin1** [dbeaver/dbeaver#20294] [Q&A]
Is the source code for extensions available anywhere? I want to test using GPT with open AI but my GPT endpoint is not openAI.com and I need to specify a custom endpoint. We have a private instance of it for enterprise use and so the endpoint url is different.

**Comment by Destrolaric** [comment]
@Austin1 Chat GPT EXtension can be found in this repository. I would recommend you to look here. Unfortunately, we have already experimented with Azure URLs, which require changing more than just URLs.
https://github.com/dbeaver/dbeaver/blob/b3f42e60feea935baab34703129d83b0d7fdd5cc/plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/gpt3/GPTCompletionEngine.java#L66
